spec/acceptance/iis_application_spec.rb

```diff
@@ -108,7 +108,7 @@
                 'bindinginformation'   => '*:443:#{site_hostname}',
                 'protocol'             => 'https',
                 'certificatestorename' => 'MY',
-                'certificatehash'      => '#{thumbprint}',
+                'certificatehash'      => '#{thumbprint.downcase}',
```

Comment:
Do we need an additional test case to prove that thumbprints are case insensitive or is this enough?  I wonder if the IIS spec says all thumbprints are lower